### ISSUE TITLE
CI for AdGem Unity SDK package

### ISSUE THREAD

**Issue by tustanivsky** [AdGem/adgem-sdk-unity-package#1]
We want to setup basic CI/CD pipeline for this repo allowing us to do the following:

1. Automatically check for updates to Unity SDK dependencies. Ideally, whenever a new AdGem Android/iOS SDK is released we want to create a corresponding PR to Unity SDK which bumps its version in the [config file](https://github.com/AdGem/adgem-sdk-unity-package/blob/ea6e7aae900bef3ec8d44bc73f68b224b5203e20/Editor/Android/AdGemDependencies.xml#L3) (see [External Dependency Manager](https://github.com/googlesamples/unity-jar-resolver) for extra context)
2. Generating new releases during the release process (need to add a semantic release configuration file `.releaserc`)
3. Pull requests title linting

**Comment by dakotawashok** [comment]
We've got semantic releases and pull request linting working, mostly. There is more work needed to get the exact file structure like how that first release is.
Your first ask is still a work in progress 🫡

**Comment by dakotawashok** [comment]
@tustanivsky I've got a GitHub Action implemented now for updating the Android Dependencies Manifest file. It hasn't gone all the way through yet, but I'll keep an eye on it. 
If this is satisfactory, feel free to close this issue. 
Do you want something similar done for the iOS dependencies? Which file is that, do you know?

**Comment by tustanivsky** [comment]
> Do you want something similar done for the iOS dependencies? Which file is that, do you know?

@dakotawashok Yes, I've created a small PR that explicitly sets the iOS dependency version in the same file so we can track the corresponding updates, too - #8.
